packages/client/src/pages/game/EndScreen/index.tsx

```diff
@@ -14,6 +15,21 @@ type EndScreenProps = {
 
 export const EndScreen: FC<EndScreenProps> = ({ value, onClick }) => {
   const dispatch = useAppDispatch();
+  const { user } = useAppSelector((state) => state.auth);
+
+  useEffect(() => {
+    const setRaiting = async () => {
+      try {
+        if (user?.login && typeof value === 'number' && value >= 0) {
+          await setRatingTeam({ login: user?.login, rating: value });
```

Comment:
_Не критично, но лучше исправить_

Тут можно убрать опциональность `user`, тк выше происходит проверка на его наличие

```suggestion
          await setRatingTeam({ login: user.login, rating: value });
```